Maple2.File.Tests/TestUtils.cs

```diff
@@ -6,7 +6,7 @@
 namespace Maple2.File.Tests;
 
 public static class TestUtils {
-    private const string m2dPath = @"C:\Nexon\Library\Library\maplestory2\appdata\Data";
+    private const string m2dPath = @"B:\Files\Maplstory 2 Client\appdata\Data";
```

Comment:
prob shouldn't be hardcoded, can we use an env var instead like how ingest does it?

Reply:
woops this slipped past me.... thanks github desktop